CMakeLists.txt

```diff
@@ -1247,6 +1264,13 @@ if (EXISTS "${HDF5_SOURCE_DIR}/c++" AND IS_DIRECTORY "${HDF5_SOURCE_DIR}/c++")
 
     include (${HDF_RESOURCES_DIR}/HDFCompilerCXXFlags.cmake)
 
+    if (HDF5_ENABLE_PARALLEL AND MPI_CXX_FOUND)
```

Comment:
This probably should go into ${HDF_RESOURCES_DIR}/HDFCompilerCXXFlags.cmake file which protects from the configuration from force-enabling the CXX language if not requested.

Reply:
I'm not sure if I understand that. The code is inside `HDF5_BUILD_CPP_LIB` condition, so I don't see how it would be enabled accidentally.

That said, I can move it. Do I understand correctly that I should move it inside the `CMAKE_CXX_COMPILER_LOADED` block?